frontend/src/App.tsx

```diff
@@ -19,9 +19,9 @@ export const App: React.FC = () => (
             <Root>
                 <Switch>
                     <Route exact path="/" component={HomeRoute} />
-                    <Route exact path="/r/:path+" component={RealmRoute} />
-                    <Route exact path="/about" component={About} />
-                    <Route exact path={["404", "*"]} component={NotFound} />
+                    <Route path="/about" component={About} />
+                    <Route path="/r/:path+" component={RealmRoute} />
```

Comment:
Why did you remove the `exact`?

Reply:
It's unnecessary. The routes are prefix-free apart from `/`, so that's the only one that needs it.

Reply:
As far as I understand, without exact, `/about/banana` will also render the `About` component. That's why I added the `exact`.

Reply:
Ah wait, without it, stuff like `/about/x` leads to the about page ... But for `/r/:path` it should be redundant. Adding it back to the others, I guess.